css/admin.css

```diff
@@ -91,10 +94,14 @@ Enqueued only within the admin screen. (Via enqueue.php)
+>>>>>>> develop
```

Comment:
Same here ... 
I noticed this because I'm getting the following error on console: 
`Error while traversing the CSS: Error: undefined:7:1: missing '{'`
and the editor styling are broken